etc/create_database_MySQL.sql

```diff
@@ -45,4 +45,5 @@ CREATE TABLE theme_assignment (
 
 INSERT INTO moodle_lrs_mapping VALUES ('MC Store', 'askabot@fakemail.de');
 INSERT INTO ACCESS VALUES ('LMSBot', 'https://moodle.tech4comp.dbis.rwth-aachen.de/course/view.php?id=18', 'askabot_webscience1');
-INSERT INTO ACCESS VALUES ('Mentoring Chatbot', 'https://moodle.tech4comp.dbis.rwth-aachen.de/course/view.php?id=20', 'askabot_uni1');
\ No newline at end of file
+INSERT INTO ACCESS VALUES ('Mentoring Chatbot', 'https://moodle.tech4comp.dbis.rwth-aachen.de/course/view.php?id=20', 'askabot_uni1');
+INSERT INTO ACCESS VALUES ('Mentoring Chatbot', 'https://moodle.tech4comp.dbis.rwth-aachen.de/course/view.php?id=37', 'askabot_uni2');
```

Comment:
These keys are necesary for the verification of the Bot with the Moodle Data Proxy when updating data from the users in Moodle